### PR TITLE
Proposal: Upgrade the IPFS All Hands Format

### PR DESCRIPTION
**Pull request by pkafei**
Changes for IPFS ALL HANDS ON CALL

### CODE REVIEW

File: IPFS_ALL_HANDS_FORMAT.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+The main purpose of the IPFS ALL HANDS ON CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use this platform to:
+
+1. Attract skilled contributors to your endeavor
+
+2. Motivate the community about adding support for a new feature
+
+3. Find new partners, capabilities, or challenges for your project to work on/with.
+
+
+The IPFS ALL HANDS ON CALL also gives you a chance to practice public speaking skills. Think of it as a [Toast Masters](https://www.toastmasters.org/) for the IPFS crowd. Below are some public speaking resources.
```
Comment (diasdavid) [inline IPFS_ALL_HANDS_FORMAT.md]: I believe the intention was to actually capture more of the content that the IPFS Community has been producing. If the intent is to get people good at public speaking skills, there are better ways to do it.

I particularly don't think that screen-sharing to 50 people + do Live Stream to hundreds of contributors is the best way to practice, specially given that there is no feedback to the talk itself. Toastmasters in the other hand has a very structured process to help people growth on this end.

File: IPFS_ALL_HANDS_FORMAT.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* Three 5 minute talks (3x per month)
+
+In total we will have a talk every Monday. The times for the talks are:
+
+| Time (PST - UTC -CET ) |
+---|
+9:00 **17:00** 18:00 |
+
+
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot com.
```
Comment (diasdavid) [inline IPFS_ALL_HANDS_FORMAT.md]: There is no such domain as `protocollabs.com`

File: IPFS_ALL_HANDS_FORMAT.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+
+| November 5th            | November 12th           | November 19th          | November 26th          |
+| ------------------------|------------------------|------------------------|-------------------------|
+| "Intro to Textile App" Sander Pick | TBD         |     TBD                |    TBD                  |
+|
+
+
+### Looking forward to all of your talk proposals!
```
Comment (diasdavid) [inline IPFS_ALL_HANDS_FORMAT.md]: It would be great to keep a list of all the previous talks/demos (there has been many) to excite folks to do their own

File: IPFS_ALL_HANDS_FORMAT.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+### IPFS All Hands Format
+
+![](https://media.giphy.com/media/JFawGLFMCJNDi/giphy.gif)
```
Comment (diasdavid) [inline IPFS_ALL_HANDS_FORMAT.md]: This gif doesn't make sense to me as the gif you want people to see when they are trying to learn "the IPFS All Hands Format"

Comment (lidel) [reply IPFS_ALL_HANDS_FORMAT.md]: https://i.imgur.com/mVQeLlo.gif ?

File: IPFS_ALL_HANDS_FORMAT.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS ON CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use this platform to:
```
Comment (momack2) [inline IPFS_ALL_HANDS_FORMAT.md]: Is there a reason the name was changed to "IPFS All Hands On Call"? Previously this was an "all hands" call - aka "call" was a descriptor of this being a sync "all hands" meeting. Now it's an all hands "on call" - which sounds like a help session or by-request office hours that doesn't seem accurate, or it's an "all hands on" call - which sounds like a workshop or something. Not sure if I missed the memo about this name change or whether it's actually just an accident... =]

**Comment by mikeal** [comment]
Should we change the name of the call to something like "IPFS Weekly
Update" or "IPFS Weekly Broadcast?"

**Comment by terichadbourne** [comment]
I agree that this should *not* be called “on call.” In common jargon, “on call” means we’re taking turns being responsible for something, which isn’t the case here. “All hands” to me means “mandatory team/department/company-wide” meeting, and It’s not clear to me whether the call is meant to be mandatory for certain employees or just strongly encouraged. 

Based on the current format of the call, I like Mikeal’s suggestion of “IPFS Weekly Update.” I’d avoid using the word broadcast unless we’re 100% fine with people skipping the call and watching the recording later. Broadcast sounds much more unidirectional and non-participatory to me.

**Comment by pkafei** [comment]
It is _"IPFS Hands On" Call_ not _IPFS Hands "On Call"_, but I prefer the change "IPFS Weekly Update" to avoid any confusion.

File: IPFS_ALL_HANDS_FORMAT.md (hunk 1 above)
Comment (meiqimichelle) [reply IPFS_ALL_HANDS_FORMAT.md]: @daviddias for the record, I had also heard the argument that changing to this format would be a great way to give our community a friendly space to test how to communicate new ideas and practice presenting generally. To put it another way, this new structure would allow us to find and encourage people who might not normally engage (to your point about capturing more of what our community has been doing), and a selling point to get people excited to do it would be the friendly presentation space and practice. Perhaps it is not quite Toastmasters, but I don't think the point is like TOTALLY out of line ;)

File: IPFS_ALL_HANDS_FORMAT.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+ ### 🥁November Scheduled Talks🥁
+
+
+
+
+| November 5th            | November 12th           | November 19th          | November 26th          |
+| ------------------------|------------------------|------------------------|-------------------------|
+| "Intro to Textile App" Sander Pick | TBD         |     TBD                |    TBD                  |
+|
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: @pkafei we need this PR finished ASAP so that folks can start using the standard way to propose sessions

File: IPFS_ALL_HANDS_FORMAT.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+### While you're here check out some of our past talks 
+
+[IPFS ALL Hands Call November 12, 2018](https://youtu.be/0fT9HC2Crqw)
+
+[IPFS ALL Hands Call November 5, 2018](https://youtu.be/4gVUKbXT0cM)
+
+[IPFS ALL Hands Call October 29, 2018](https://youtu.be/o1tUzgThZK0)
+
+[IPFS ALL Hands Call October 22, 2018](https://youtu.be/iEagJfB8AvM)
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: It would be useful if these links would point to the timecode of the start of the talk and also the title was the title of the talk vs the all hands date.

Example

`[Mozilla/libdweb by Irakli Gozalishvili](https://youtu.be/0fT9HC2Crqw?t=132)`


You can get these timecodes by clicking the share button and pinpoint the moment you want it to start:
image

File: IPFS_ALL_HANDS_FORMAT.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+4. Practice speaking in front of a captivated audience
+
+
+## The Format
+
+Every month we will have:
+
+* 15 minute talk from guest presenter (3x per month)
+* 5 minute talks lightening talks (1x per month)
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: typo: lightning

File: IPFS_ALL_HANDS_FORMAT.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+In total we will have a talk every Monday. The times for the talks are:
+
+| Time (PST - UTC -CET ) |
+---|
+9:00 **17:00** 18:00 |
+
+
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot ai.
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: @pkafei the email is still incorrect. the correct domain is just @protocol.ai, therefore your email address is portia@protocol.ai

File: IPFS_ALL_HANDS_FORMAT.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+
+
+
+## List of Upcoming Talks  ⚡️📅 ⚡️
+![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1542475278/ipfs_calendar_dyxtst.jpg)
+
+
+
+_[Sign up here if you would like to speak!](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)_
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: The protocol for signing up is not clear enough. Looks like that spreadsheet is the schedule already. Should folks put themselves for a date? After they add themselves, how can they be sure that the proposal was received and scheduled?

File: IPFS_ALL_HANDS_FORMAT.md (hunk 1 above)
Comment (daviddias) [reply IPFS_ALL_HANDS_FORMAT.md]: In order to make the All Hands a very useful for a public speaking training, then we can take a lot of inspiration from Toastmasters. In Toastmasters they always have a time keeper, grammarian, "Ah" counter, evaluator and at the end, everyone can give feedback to the talk.

Comment (momack2) [reply IPFS_ALL_HANDS_FORMAT.md]: To David's point, while I agree we should be a friendly, welcoming venue for all kinds of presenters, I don't think we should be pitching this call as a toastmasters or incorporating toastmaster-style feedback into the format. I think the majority of our community is attending for the content communicated about improvements to IPFS or applications built on top - and don't want to distract from that communication channel with meta-level speaking feedback that might dissuade high-leverage community members from engaging or seeing this call as a valuable space to present their work. 

Perhaps we could rephrase this as "... also gives you a chance to practice communicating your IPFS project or to preview talks for other venues with a friendly and supportive audience."

Comment (pkafei) [reply IPFS_ALL_HANDS_FORMAT.md]: I can see how the association with Toastmasters can be unhelpful and I think the rephrasing _"...also gives you a chance to practice communicating your IPFS project or to preview talks for other venues with a friendly and supportive audience."_ probably is more accurate reflection of how speakers can use IPFS All Hands meeting as a space to practice public speaking.

File: IPFS_ALL_HANDS_FORMAT.md (hunk 7 above)
Comment (daviddias) [reply IPFS_ALL_HANDS_FORMAT.md]: @pkafei did you noticed that I listed the name of the talk and the speaker vs just saying "IPFS All Hands" in the example above?

File: IPFS_ALL_HANDS_FORMAT.md [hunk 11]
```diff
@@ -45,18 +45,36 @@ If you have a talk you want to give, [sign up here](https://docs.google.com/spre
 ![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1542475278/ipfs_calendar_dyxtst.jpg)
 
 
+## Steps to signing up and speaking
 
-_[Sign up here if you would like to speak!](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)_
+1. Write a short blurb about your talk and a proposed date for your talk. Send it to portia @ protocol dot com. You should receive an update about the status of your talk within 24-48 hours.
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: @pkafei you got your email wrong again. I'm starting to think that you are just trolling me :)

File: IPFS_ALL_HANDS_FORMAT.md [hunk 12]
```diff
@@ -36,7 +36,7 @@ In total we will have a talk every Monday. The times for the talks are:
 9:00 **17:00** 18:00 |
 
 
-If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot ai.
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol dot ai.
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: This paragraph should link to the steps below, which are more complex than just adding ourselves to a spreadsheet.

File: IPFS_ALL_HANDS_FORMAT.md [hunk 13]
```diff
@@ -45,18 +45,36 @@ If you have a talk you want to give, [sign up here](https://docs.google.com/spre
...
+
+### If you're looking for more inspiration, check out some of our past talks! 
+
+[IPFS ALL Hands Call November 12, 2018](https://youtu.be/0fT9HC2Crqw?t=116)
+
+[IPFS ALL Hands Call November 5, 2018](https://youtu.be/4gVUKbXT0cM?t=52)
+
+[IPFS ALL Hands Call October 29, 2018](https://youtu.be/o1tUzgThZK0?t=18)
+
+[IPFS ALL Hands Call October 22, 2018](https://youtu.be/iEagJfB8AvM?t=124)
```
Comment (daviddias) [inline IPFS_ALL_HANDS_FORMAT.md]: It would be better if these were actual the talk tiles and speakers rather than just saying `IPFS All Hands Call`

File: IPFS_ALL_HANDS_FORMAT.md (hunk 12 above)
Comment (pkafei) [reply IPFS_ALL_HANDS_FORMAT.md]: I had second thoughts about the spreadsheet and I wondering if it would be a better experience if folks just emailed their talk ideas to me and I'll add it to the spreadsheet myself.

File: IPFS_ALL_HANDS_FORMAT.md (hunk 13 above)
Comment (pkafei) [reply IPFS_ALL_HANDS_FORMAT.md]: We didn't start having talks until November. I'm going to leave `IPFS ALL Hands Call` title for Youtube videos that used the old format.

**Comment by terichadbourne** [comment]
@pkafei  authorized me to go ahead and make a few grammar edits directly, and I've also edited a couple of sections lightly for clarity in this commit. Note that I've updated "IPFS All Hands" to "IPFS Weekly Update" as previously discussed, which also changes the filename and the reference in the main README.md file.

**Comment by flyingzumwalt** [comment]
Question about the change in name -- to me "Weekly Update" implies an update about the whole project, which this isn't. These calls are formatted to focus on a particular person's work or a particular endeavor on each call, with only a minimal amount of announcements. To me that doesn't sound like an IPFS Update as much as an IPFS Community Call, IPFS Ideas Exchange, or just IPFS Weekly Presentation.

If I'm the only one who doesn't like "Weekly Update", I'm fine with it. I'll just think of it like SNL's Weekend Update.

**Comment by terichadbourne** [comment]
I'm fine with "IPFS Weekly Update," but if others feel as @flyingzumwalt does then I'd propose considering using the words _spotlight_ or _highlight_ to substitute for _update_ since it will convey the narrower focus, as in these potential options... 

IPFS Weekly Spotlight
IPFS Community Spotlight
IPFS Weekly Highlights
IPFS Community Hightlights

**Comment by daviddias** [comment]
Why not keep it as `IPFS Weekly Call`, it happens weekly, it is a Call, people discuss and present things. 🚲🏡

File: IPFS_WEEKLY_CALL_FORMAT.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1541176158/ipfs_crew_xzbhxr.png)
+
+## List of upcoming talks  ⚡️📅⚡️
+
+[View the list of scheduled talks](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)
+
+## Steps to signing up and speaking
+
+- 1. Write a short blurb about your talk and propose a date, consulting the [calendar of upcoming talks](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) to see what dates are available. Email your proposal to portia @ protocol dot ai and you should receive an update about the status of your talk within 24-48 hours.
```
Comment (daviddias) [inline IPFS_WEEKLY_CALL_FORMAT.md]: Let's actually use a mlist for this proposals which Portia and others can subscribe to. Let me get that done

Comment (daviddias) [reply IPFS_WEEKLY_CALL_FORMAT.md]: created `ipfs-weekly-call-rfp@ipfs.io`

**Comment by daviddias** [comment]
I believe this is now in a good enough state to ship.
